Add contributing documentation (#1171)
hitobito/hitobito: CONTRIBUTING.md

> # How to contributing to hitobito

Reviewer: ```suggestion
# How to contribute to hitobito
```

First off, thank you for considering contributing to hitobito. hitobito is made for and by its own community. Happy to welcome you.

Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.

> hitobito is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, from expand the documentation, submitting bug reports, feature requests, comment issues or writing code.

Reviewer: ```suggestion
hitobito is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, be it expanding the documentation, submitting bug reports, feature requests, comment issues or writing code.
```

Currently, the rest of this document is only available in German. Translations are welcome :man_dancing:

# Ground Rules

* Nicht nur der Source Sode von hitobito ist offen, sondern auch die Menschen, welche mit und an hitobito arbeiten.
* Damit wir alle noch lange Freude an hitobito haben, wünschen wir uns eine gute Qualität.

> * Auch kleine Anpassungen sind willkomen.

Reviewer: ```suggestion
* Auch kleine Anpassungen sind willkommen.
```
